source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -217,7 +218,7 @@ def output(self, key, obj):
             "administrative_regions":
                 create_administrative_regions_field(geocoding) or create_admin_field(geocoding),
             "properties": {p.get('key'): p.get('value') for p in geocoding.get("properties", [])},
-            "address": create_address_field(geocoding.get("address"))
+            "address": create_address_field(geocoding.get("address"), lat, lon)
```

Comment:
I might be paranoiac, but I will prefer that we use named parameter for lat and lon, as: 
```
create_address_field(geocoding.get("address"), lat=lat, lon=lon)
```